### PR TITLE
Setup global document in node land

### PR DESCRIPTION
**Pull request by samccone**
R: @mounirlamouri

### CODE REVIEW

File: manifest-parser.js [hunk 1]
```diff
@@ -0,0 +1,296 @@
+var ManifestParser = (function() {
```
Comment (mounirlamouri) [inline manifest-parser.js]: The file moved to static/manifest-parser.js, shouldn't you modify that one instead?

File: test/manifest-parser-tests.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
+var ManifestParser = require('../manifest-parser.js');
```
Comment (mounirlamouri) [inline test/manifest-parser-tests.js]: Did you mean to delete the other one?

**Comment by mounirlamouri** [comment]
Looks great. Thanks! :)

**Comment by samccone** [comment]
thanks for the great review @mounirlamouri :clap:

**Comment by samccone** [comment]
@mounirlamouri mind cutting a new npm release?

**Pull request merged**